### PR TITLE
Wagtail upgrade 2.13

### PR DESCRIPTION
**Pull request by rcoldiron**
Thank you for contributing! Please follow the guidelines below to submit your
pull request. Additional details are available in our
[Contributor Guide](https://docs.coderedcorp.com/cms/stable/contributing/index.html).

#### Description of change
Upgrade wagtail to 2.13

Made a slight css change for admin but didn't notice anything else looking weird. 

#### Tests
Tested manually against a few sites and no problems were noticed.

### CODE REVIEW

File: coderedcms/static/coderedcms/css/codered-admin.css [hunk 1]
```diff
@@ -255,6 +255,9 @@ input[type='checkbox']::before, input[type='radio']::before {
     .nav-main .footer-submenu {
         overflow: hidden;
     }
+    .footer .footer-submenu {
```
Comment (vsalvino) [inline coderedcms/static/coderedcms/css/codered-admin.css:258]: Could this be added to the rule above (line 255) - just to keep the rules to a minimum?

**Comment by rcoldiron** [comment]
Toggle up arrow for Admin sub-menu doesn't work. I didn't find the js that's supposed to do it in your js files.

**Comment by goodtiding5** [comment]
Please check my pull request.  It fixed the Admin submenu.

**Comment by vsalvino** [comment]
Tested locally and everything looks good!

**Pull request merged**